ecs-microservice/main.tf

```diff
@@ -42,12 +48,13 @@ module "ecs_fargate_microservice" {
 
 
 resource "aws_lb_listener_rule" "ecs_fargate_microservice_lb_listener" {
-  listener_arn = var.alb_http_listener.arn
+  for_each     = var.lbs
+  listener_arn = each.value.listener_arn
   priority     = var.alb_priority
```

Comment:
dette burde tryne, men gjør det kanskje ikke fordi regelen blir opprettet på forskjellige lastbalanserere?

Noen sjanse for at vi bare kan fjerne priority?
Den er optional fra ressursen sin side, og får et auto-assignet nummer mellom 1 og 5000

Reply:
Korrekt: det tryner ikke fordi det er forskjellige lastbalanserere.
Hadde ikke lyst å mene noe om ting jeg ikke _måtte_ mene noe om i denne PRen så derfor har jeg ikke tatt stilling til å fjerne priority :)